src/marqo/core/unstructured_vespa_index/unstructured_document.py

```diff
@@ -23,7 +23,7 @@ class UnstructuredVespaDocumentFields(MarqoBaseModel):
     score_modifiers_fields: Dict[str, Any] = Field(default_factory=dict, alias=unstructured_common.SCORE_MODIFIERS)
     vespa_chunks: List[str] = Field(default_factory=list, alias=unstructured_common.VESPA_DOC_CHUNKS)
     vespa_embeddings: Dict[str, Any] = Field(default_factory=dict, alias=unstructured_common.VESPA_DOC_EMBEDDINGS)
-    vespa_multimodal_params: Dict[str, str] = Field(default_factory=str,
+    vespa_multimodal_params: Dict[str, str] = Field(default_factory=dict,
```

Comment:
I'll change it back to avoid introducing new regressions. Will change it to use the correct default value (or make it optional) in the next refactoring.

Reply:
Your fix is correct. It was a mistake I made before. It is working because there is a check [here](https://github.com/marqo-ai/marqo/blob/fa0d695f7b03227caad6aab5fbed7b79b4dc06b0/src/marqo/core/unstructured_vespa_index/unstructured_document.py#L190) so even if it's an empty string there isn't any issue. It's fine that you leave it as it is and I will fix it.